doc/en/components/grids/_shared/state-persistence.md

```diff
@@ -310,32 +554,51 @@ public activeTemplate = (ctx: IgcCellTemplateContext) => {
 }
 ```
 
-<!-- ComponentEnd: TreeGrid  -->
-
-1. Query the template view in the component using @ViewChild or @ViewChildren decorator. In the `ColumnInit` event handler, assign the template to the column `BodyTemplate` property:
+<!-- ComponentEnd: HierarchicalGrid  -->
 
+<!-- Angular -->
+2. Query the template view in the component using @ViewChild or @ViewChildren decorator. In the `ColumnInit` event handler, assign the template to the column `BodyTemplate` property:
+<!-- end: Angular -->
 
 ```typescript
 @ViewChild('activeTemplate', { static: true }) public activeTemplate: TemplateRef<any>;
 public onColumnInit(column: IgxColumnComponent) {
     if (column.field === 'IsActive') {
         column.bodyTemplate = this.activeTemplate;
-        column.summaries = MySummary;
-        column.filters = IgxNumberFilteringOperand.instance();
     }
 }
 ```
 
-```razor
-Add blazor handler for bodyTemplate
-```
+<!-- Blazor, React, WebComponents -->
+2. In the `ColumnInit` event handler, assign the template to the column `BodyTemplate` property:
+<!-- end: Blazor, React, WebComponents -->
 
 ```typescript
-public onColumnInit(column: IgcColumnComponent) {
+public onColumnInit(event: any) {
+    const column = event.detail as IgcColumnComponent;
+    if (column.field === 'IsActive') {
+        column.bodyTemplate = this.activeTemplate;
```

Comment:
Does changing the template runtime work? I think there way a bug logged about this at some point.  If it doesn't work, we probably won't be able to add a fix for this release so maybe we should just leave it as a  limitation and not show it in the docs.